platform/pom.xml

```diff
@@ -27,7 +27,7 @@
     <dependencyManagement>
         <dependencies>
             <dependency>
-                <artifactId>karaf</artifactId>
+                <artifactId>karaf-bom</artifactId>
```

Comment:
Finally, a karaf bom 🎉 

This should get moved up to the root pom's dependency management, and there's probably a lot of cleanup that could happen. For example, we may not need to manage versions for certain Karaf dependencies anymore since they're declared in the bom. This (https://github.com/codice/ddf/pull/6542/files#diff-6fc3fde0bdc5a8c8c3c0273db04a6b47b6728ac241dd3244ebb7d5ffb5345146R159-R164) 
``` 
<dependency>
    <groupId>org.apache.felix</groupId>
    <artifactId>org.apache.felix.framework</artifactId>
    <version>${felix.framework.version}</version>
    <scope>test</scope>
</dependency>
```
can change to
```
<dependency>
    <groupId>org.apache.felix</groupId>
    <artifactId>org.apache.felix.framework</artifactId>
    <scope>test</scope>
</dependency>
```
and then we can get rid of the `felix.framework.version` property in the root pom.